files/motd/template.md

```
@@ -1,17 +1,19 @@
-# 🦖 Welcome to Dakota
+# 🦖 Dakota · Field Dispatch
 
-󱋩 `${MOTD_IMAGE_NAME}:${MOTD_IMAGE_TAG}`
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)